src/utils/api.ts

```diff
@@ -0,0 +1,51 @@
+import { StocksType } from '../types';
+
+export async function postSearchResult(symbol: string): Promise<StocksType> {
+  const response = await fetch('/api/stocks', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ symbol: symbol }),
+  });
+  if (!response.ok) {
+    const errorMessage = await response.text();
+    throw errorMessage;
+  }
+  const result: StocksType = await response.json();
+  return result;
+}
+
+export async function deleteSingleStock(symbol: string): Promise<StocksType> {
+  const response = await fetch('/api/stocks', {
+    method: 'DELETE',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ symbol: symbol }),
+  });
+  if (!response.ok) {
+    const errorMessage = await response.text();
+    throw errorMessage;
+  }
+  const result: StocksType = await response.json();
+  return result;
+}
+
+// export async function postLoginUser(
+//   user: Partial<UserType>
+// ): Promise<UserType> {
+//   const response = await fetch('/api/users/login', {
+//     method: 'POST',
+//     headers: {
+//       'Content-Type': 'application/json',
+//     },
+//     body: JSON.stringify(user),
+//   });
+//   if (!response.ok) {
+//     const errorMessage = await response.text();
+//     throw errorMessage;
+//   }
+//   const result: UserType = await response.json();
+//   return result;
+// }
```

Comment:
Why is this here?

Reply:
I wanted to build the user login and had started on the basis of a PR. Then I decided against it, because I had prioritized other things and simply forgot to delete this part.